tests/kafkatest/services/verifiable_producer.py

```diff
@@ -180,6 +184,9 @@ def pids(self, node):
     def alive(self, node):
         return len(self.pids(node)) > 0
 
+    def clean_shutdown(self, node):
```

Comment:
Same comment as in console consumer - don't need this method, and `nodes_clean_shutdown` should have a slightly different name